cmake/FindELPA.cmake

```diff
@@ -7,34 +7,46 @@
 #  ELPA_INCLUDE_DIR - Where to find ELPA headers.
 #
 
-find_path(ELPA_INCLUDE_DIR
+find_package(PkgConfig)
```

Comment:
Please also update user docs for utilizing `pkgconfig`

Reply:
OK. Is it necessary to add something about installing pkg-config?

Reply:
> OK. Is it necessary to add something about installing pkg-config?

@yizeyi18 Yes, please

Reply:
Sure. Updated.